pkg/datapath/loader/base.go

```diff
@@ -293,6 +332,41 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 	log.Infof("Setting up base BPF datapath (BPF %s instruction set, %s clock source)",
 		args[initBPFCPU], clockSource[option.Config.ClockSource])
 
+	// EKS expects the aws daemonset to set up the following rules that
```

Comment:
> Is this the best location to add the rules? I initially considered the AWS-related code for the cilium-cni but I don't think this is used to manage the node itself.

I think it's spread over multiple places likely including `bpf/init.sh` right now, so I'm not sure there's a "wrong" answer. At some point we should converge them, but that's a longer-term objective.